app/views/employees/_form.html.erb

```diff
@@ -13,13 +13,13 @@
 
 <div class="field-group field-group--two-cols">
     <div class="field field--required">
-        <%= f.label :last_name, class: "field__label" %>
-        <%= f.text_field :last_name, classs: "field__input" %>
+        <%= f.label :surname, class: "field__label" %>
+        <%= f.text_field :surname, classs: "field__input" %>
     </div>
 
-    <div class="field">
-        <%= f.label :other_names, class: "field__label" %>
-        <%= f.text_field :other_names, classs: "field__input" %>
+    <div class="field field--required">
+        <%= f.label :forenames, class: "field__label" %>
+        <%= f.text_field :forenames, classs: "field__input" %>
```

Comment:
Can you see if you can make the text field HTML required? I see we don't currently do that for any of our required fields (we only add the CSS class to make it look required) but it would be good to add HTML level validation too.